crates/swc/tests/exec/issues-4xxx/4406/1/exec.js

```diff
@@ -0,0 +1,18 @@
+async function* asyncGenerator(
+) {
+    for (let i = 5; i < 10; i++) {
+        yield new Promise((res) => setTimeout(() => res({ success: true }), 5));
+    }
+    yield { success: true };
+}
+
+async function* execute(
+) {
+    yield* asyncGenerator();
+}
+
+(async () => {
+    for await (const p of execute()) {
+        console.log(await p);
+    }
+})();
```

Comment:
New to this repo, but isn't it #4405 (not #4406)?

Reply:
You are right, thank you!